src/components/PromiseList/PromiseCard.svelte

```diff
@@ -1,13 +1,20 @@
 <script lang="ts">
 	import { formatThaiDate } from '$lib/date-parser';
-	import { PromiseStatus, type PromiseSummary } from '$models/promise';
 	import Quotes from 'carbon-icons-svelte/lib/Quotes.svelte';
+	import { PromiseStatus } from '$models/promise';
+
+	export let id: string;
+	export let status: PromiseStatus;
+	export let latestProgressDate: Date | undefined = undefined;
+	export let party: { name: string; logo: string };
+	export let statements: string[];
+	export let keywords: string[];
+	export let categories: string[];
 
-	export let promiseSummary: PromiseSummary;
 	export let isList: boolean;
```

Comment:
This is not a part of your issue, but I think we can improve it. Boolean props like this should be optional with `false` as a default value like `export let isList = false;`.

So that a component who use it can write like:

```html
<PromiseCard isList /><!- equal to isList={true}  ->
<PromiseCard /><!- equal to isList={false}  ->
```

Treating boolean prop as a flag I think will make the component easier to read. Then you can update the usage in story file.